### ISSUE TITLE
Feature Request - Airplay Main SCREEN

### ISSUE THREAD

**Issue by caryhon139** [ElvishArtisan/rivendell#715]
I miss the horizontal VU meter in the new 4.0. Perhaps making it an option. Also, the Op Guide must still be in development but.. there is no reference to the segment titled "User Logo". The message panel to the left of that segment takes up too much space while other segments could use more space for useful material. Just my opinion.

**Comment by ElvishArtisan** [comment]
> [T]he Op Guide must still be in development but.. there is no reference to the segment titled "User Logo". 

Correct. The Ops Guide will be updated in due course as we move from beta to release candidate versions. The next beta release should see the ability sto pecify the image to use there in ```rdadmin(1)```.

> The message panel to the left of that segment takes up too much space while other segments could use more space for useful material.

That's quite site-specific. Bear in mind that that meter strip could potentially need to hold up to nine meters.

**Comment by caryhon139** [comment]
Perhaps an option for those of us doing simpler things and don't require more than 2. Option on installation or a spot on rd,conf which could be changed should the user need to add more meters.  Just a thought. Appreciate your efforts.

**Comment by fpeyraud** [comment]
OK. For M5 missing, I've just realized it is because it's affected to a virtual log, so that's OK.
